src/headerdef.rs

```diff
@@ -61,4 +73,22 @@ mod tests {
 
         assert_eq!(HeaderDef::_TestHeader.get_headername(), "test-header");
     }
+
+    #[test]
+    /// Test that headers are parsed case-insensitively
+    fn headerdef_case() {
+        let (headers, _) =
+            mailparse::parse_headers(b"fRoM: Bob\naUtoCryPt-SeTup-MessAge: v99").unwrap();
+        assert_eq!(
+            headers
+                .get_headerdef(HeaderDef::AutocryptSetupMessage)
+                .unwrap(),
+            Some("v99".to_string())
+        );
+        assert_eq!(
+            headers.get_headerdef(HeaderDef::From_).unwrap(),
```

Comment:
i think it should be `get_header_value` or `get_headervalue` -- as we are not returning a def but the value that the def is defining :)